base_multi_image/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
I respected the original headers in the previous versions. Removing copyrights is a bit disrespectful at my sight.

However new files I usually just put mine, since they are totally new. Anyway, the CLA shares the copyright with OCA and grants them rights to do whatever, so this is mostly anecdotal.

Anybody wishing to know the real contributors can do so by `git blame`, line by line, and as long as they are listed in the README, seems legit to me.